content/authoring/kata/adding-images.md

```diff
@@ -0,0 +1,77 @@
+---
+kind: recipe
+---
+
+# Adding Images to Kata Descriptions
+
+Currently, it's not possible to attach any files or additional assets to a kata, but many authors would like to illustrate their descriptions with images. There are plans to introduce the possibility of uploading and attaching files to descriptions, but until it's implemented, authors have to resort to other techniques. There are a few of them, and each of them has its advantages and disadvantages.
```

Comment:
Before KaTeX was added, authors used to use images to show formulas. Maybe add a note mentioning it's no longer necessary and link to the reference.

```markdown
:::note
// something like: if you're showing math formulas, use katex instead
:::
```

I haven't looked into how many kata are still doing that, but those should be updated to use KaTeX. If it's not possible for some reason, we should discuss what to do.